src/Http/RequestContext.php

```diff
@@ -37,11 +37,15 @@ class RequestContext implements Context
     /** @var RequestFactoryInterface */
     private $requestFactory;
 
-    public function __construct(PluginClientBuilder $builder, StreamFactoryInterface $streamFactory, RequestFactoryInterface $requestFactory)
+    /** @var string */
+    private $graphqlEndpoint;
```

Comment:
rather than a propoerty, it would be better IMO to have a `I create the following graphql request on :graphqlEndpoint::` or something like that

Reply:
Yes, but as an graphql endpoint is something that is fixed the same for all graphql request, it is redundant to precise it again in each request in my opinion

Reply:
Ping ?